include/class.forms.php

```diff
@@ -2198,6 +2198,9 @@ function toString($value) {
         if (is_array($value))
             return '';
 
+        if (is_array($value))
+            return '';
+
```

Comment:
Despite that I don't have any idea where this code is used, but does it make sense to `return '';` twice?

Reply:
Most definitely unnecessary - fixed. Thanks